src/pinning.js

```diff
@@ -16,19 +17,22 @@ const IPFS_OPTIONS = {
   *  Pinning - a class for pinning orbitdb stores of 3box users
   */
 class Pinning {
-  constructor (cache, ipfsPath, orbitdbPath, analytics) {
+  constructor (cache, ipfsConfig, orbitdbPath, analytics, orbitCacheOpts, runCacheServiceOnly) {
     this.cache = cache
-    this.ipfsPath = ipfsPath
+    this.ipfsConfig = ipfsConfig
     this.orbitdbPath = orbitdbPath
     this.openDBs = {}
     this.analytics = analytics
+    this.orbitCacheOpts = orbitCacheOpts
+    this.runCacheServiceOnly = runCacheServiceOnly
   }
 
   async start () {
     this.ipfs = await this._initIpfs()
     const ipfsId = await this.ipfs.id()
     console.log(ipfsId)
-    this.orbitdb = new OrbitDB(this.ipfs, this.orbitdbPath)
+    const orbitOpts = this.orbitCacheOpts ? { cache: orbitDBCache(this.orbitCacheOpts) } : { }
+    this.orbitdb = new OrbitDB(this.ipfs, this.orbitdbPath, orbitOpts)
```

Comment:
The path is not used in our orbit-db-cache impl right? Makes sense to include it here though if someone where to use a custom cache though.

Reply:
@oed  were you saying pass it through to orbitDBCache? i guess, but there is no config here for generally passing in a cache, and only two caches exist so far, the orbit default and this one. If not orbit redis path is given, then it fallbacks to the default

Reply:
nvm this comment, I think I misread the code.